packages/aws-fargate/test/esm/test.js

```diff
@@ -0,0 +1,188 @@
+/*
+ * (c) Copyright IBM Corp. 2023
+ */
+
+'use strict';
+
+const { expect, assert } = require('chai');
+const { fail } = assert;
+const path = require('path');
+const constants = require('@instana/core').tracing.constants;
+
+const Control = require('../Control');
+const { expectExactlyOneMatching } = require('../../../core/test/test_util');
+const config = require('../../../serverless/test/config');
+const retry = require('../../../serverless/test/util/retry');
+
+const downstreamDummyPort = 4567;
+const downstreamDummyUrl = `http://localhost:${downstreamDummyPort}/`;
+const region = 'us-east-2';
+const account = '555123456789';
+const instrumentedContainerName = 'nodejs-fargate-test-container';
+const taskArn = `arn:aws:ecs:${region}:${account}:task/55566677-c1e5-5780-9806-aabbccddeeff`;
+const instrumentedContainerId = `${taskArn}::${instrumentedContainerName}`;
+
+const containerAppPath = path.join(__dirname, './app.mjs');
+const instanaAgentKey = 'aws-fargate-dummy-key';
+
+const requestHeaders = {
+  'X-Entry-Request-Header-1': 'entry request header value 1',
+  'X-Entry-Request-Header-2': ['entry', 'request', 'header', 'value 2'],
+  'X-Entry-Request-Header-3': 'not configured to capture this',
+  'X-Entry-Request-Header-4': ['not', 'configured', 'to', 'be', 'captured']
+};
+
+function prelude(opts = {}) {
+  this.timeout(config.getTestTimeout());
+  this.slow(config.getTestTimeout() / 2);
+  opts.platformVersion = opts.platformVersion || '1.3.0';
+  if (opts.startBackend == null) {
+    opts.startBackend = true;
+  }
+
+  let env = {
+    INSTANA_EXTRA_HTTP_HEADERS:
+      'x-entry-request-header-1; X-ENTRY-REQUEST-HEADER-2 ; x-entry-response-header-1;X-ENTRY-RESPONSE-HEADER-2 , ' +
+      'x-eXit-Request-Header-1; X-EXIT-REQUEST-HEADER-2 '
+  };
+  if (opts.env) {
+    env = {
+      ...env,
+      ...opts.env
+    };
+  }
+
+  const controlOpts = {
+    ...opts,
+    env,
+    containerAppPath,
+    downstreamDummyPort,
+    downstreamDummyUrl,
+    instanaAgentKey
+  };
+  if (opts.proxy) {
+    controlOpts.env.INSTANA_ENDPOINT_PROXY = opts.proxy;
+  }
+  return new Control(controlOpts).registerTestHooks();
```

Comment:
TBD: With your update, we are now not using the `--experimental-loader` at all?

Reply:
Updated the code, could you please check again